docs/architecture/third-party/drupal/layout-architecture.md

```diff
@@ -1 +1,6 @@
--- TO BE DEFINED --
\ No newline at end of file
+# Layout architecture
+
+The OpenEuropa project strictly follows the guidelines put forward by the [Europa Component Library (ECL)](https://ec.europa.eu/component-library).
```

Comment:
Can we change the ecl link with https://github.com/ec-europa/europa-component-library?

Reply:
Why?

Reply:
Response on the previous comment.